libs/components/core/src/lib/modules/content-info-provider/content-info-provider.ts

```diff
@@ -0,0 +1,29 @@
+import { Observable, ReplaySubject } from 'rxjs';
+
+import { SkyContentInfo } from './content-info';
+
+/**
+ * @internal
+ * An API to provide information about a parent component's content to child components.
+ * For example, toolbar can use this to provide its child components with a list
+ * descriptor they can use to construct aria labels, or tree view can provide the node
+ * name to its context menus.
+ */
+export class SkyContentInfoProvider {
+  #contentInfo: ReplaySubject<SkyContentInfo> = new ReplaySubject(1);
+  #currentValue: SkyContentInfo = {};
+
+  public patchContentInfo(value: SkyContentInfo): void {
```

Comment:
This could just be `patch` or `patchInfo` since the `Content` portion is implied both by the class name and the parameter name. I'd also change `getContentInfo` to `getInfo` to match.